repositories.yaml

```diff
@@ -834,40 +834,20 @@ applications:
       - glean/metrics.yaml
     ping_files:
       - glean/pings.yaml
-    dependencies:
-      - glean-js
+    dependencies: []  # Empty. Dependencies are set per channel.
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 180
     channels:
       - v1_name: mozilla-vpn
         app_id: mozillavpn
-        app_channel: release
-
-  - app_name: mozilla_vpn_android
-    canonical_app_name: Mozilla VPN (Android)
```

Comment:
I think to keep using this "Mozilla VPN (Android)" name for the Android app channel you can add a `description` to the channel to be used in place of the `canonical_app_name` (this is based on [me doing that in my abandoned PR doing this same thing](https://github.com/mozilla/probe-scraper/pull/469/commits/1f84e960d3faef15a3e6d2d128ae48c701a710bb#diff-a039c9b63812e8c69d4ad7a53ccb7c5b15f3721aadca48e869ca5653fa7b4229R827), where I tried to minimize the resulting changes).